frontend-instructor/src/components/addCourse/addCourse.jsx

```
@@ -0,0 +1,347 @@
+    <div className="p-6 max-w-[800px] mx-auto">
+      <h1 className="text-3xl font-bold text-center">Add New Course</h1>
+
+      <Formik
+        initialValues={initialValues}
+        validate={validateForm}
+        onSubmit={handleSubmit}
+      >
+        {({ values, errors, touched, isSubmitting, status }) => (
+          <Form>
+            {/* Error message if API call fails */}
+            {status && status.error && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)